ddtrace/contrib/pytest/plugin.py

```diff
@@ -25,6 +26,13 @@
 log = get_logger(__name__)
 
 
+def encode_test_parameter(parameter):
```

Comment:
we probably want to cache this as I can't imagine the `re.sub` is cheap to do. Given it's the pytest integration this would probably be a premature optimization though so ok deferring it for now.